_pages/2024/shared-tasks/simultaneous.md

```diff
@@ -13,40 +13,172 @@ if you want to have a line break either put a blank line in between the text or
+
+### MuST-C v2.0 tst-COMMON
+
+The participants are required to submit the output of the system (the `--output` from `simuleval`) on MuST-C v2.0 tst-COMMON.
+The segmented data can be downloaded [here](https://dl.fbaipublicfiles.com/simultaneous_translation/iwslt2023/must-c_v2.0_tst-COMMON.tgz)
+The latency constraint will be check on the MuST-C v2.0 tst-COMMON set.
+
+### Readme
+
+A `readme` file should contains the following information
+
+- Instruction to run the system on each track and language direction.
+- Results on MuST-C v2.0 tst-COMMON of the default system.
+- Any other information needed to run the system
+
+### Example
+
+An example of training, evaluating and submission preparation with the baseline systems can be found [here](https://github.com/facebookresearch/fairseq/tree/iwslt2023/examples/simultaneous_translation)
 
 
 ## Evaluation
 
-<!-- Description of metrics used for evaluation, what the official ranking is based on, links to evaluation scripts -->
+The evaluation implementation will use the latest [SimulEval](https://github.com/facebookresearch/SimulEval) toolkit.
+We will use the new [remote evaluation](https://simuleval.readthedocs.io/en/latest/tutorials/remote_evaluation.html) mode.
+
+### Metrics
+The system's performance will be evaluated in two ways:
+
+- Translation quality metrics:
+  - BLEU
+  - BLASER
+- Translation latency:
+  - Average Lagging
+  - Length Adaptive Average Lagging
+  - Average Token Delay
+  - Average Proportion
+  - Differentiable Average Lagging
+
+For latency measurement, we will contrast computation aware and non computation aware latency metrics.
+See the [SimulEval description](https://arxiv.org/abs/2007.16193) for how those metrics are defined.
+Note that the definition of average lagging has been modified from the [original definition](https://www.aclweb.org/anthology/P19-1289/) (see section 3.2 in the [SimulEval description](https://arxiv.org/abs/2007.16193)).
+
+For speech-to-speech translation, we will run the latency metrics on aligned transcripts.
 
+We will run the systems on the segmented blind test set.
+The latency is calculated on detokenized word level for En-De systems and character level for En-Zh and En-Ja systems.
+
+### Ranking
+
+The systems will be ranked by the translation quality within a latency constraint.
+The latency constraint is determined by the latency on MuST-C v2.0 tst-COMMON set.
+The constraint is determined by the average lagging,
+but all other latency metrics will be reported.
+Based on analysis on the quality-latency tradeoffs for the baseline systems,
+for all three languages, the thresholds are set as follows:
+
+- Speech-to-Text Translation: 2 seconds (Average Lagging)
+- Speech-to-Speech Translation: 2.5 seconds (Starting Offset)
 
 ## Organizers
 
-<!-- List of organizers' names and affiliations -->
+- Katsuhito Sudoh (NAIST)
```

Comment:
From the discussion yesterday, maybe we can stick to a shorter list for now. I am following up with Shinji on whether he is interested in organizing this year.

```
(to be updated)
- Katsuhito Sudoh (NAIST)
- Ondřej Bojar (Charles University)
- Xutai Ma (Meta)
- Juan Pino (Meta)
```